lib/elixir/test/elixir/protocol_test.exs

```diff
@@ -143,11 +147,23 @@ defmodule ProtocolTest do
   end
 
   test "protocol defines callbacks" do
-    assert [{:type, 13, :fun, args}] = get_callbacks(@sample_binary, :ok, 1)
-    assert args == [{:type, 13, :product, [{:user_type, 13, :t, []}]}, {:type, 13, :boolean, []}]
+    assert [{:type, 17, :fun, args}] = get_callbacks(@sample_binary, :ok, 1)
```

Comment:
Is it alright to modify these test in this way? It wasn't clear to me how the types in the callback are represented in the abstract format, and whether these values are significant.